cmd/agent/app/remote_config.go

```diff
@@ -0,0 +1,124 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+package app
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/api/security"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/proto/pbgo"
+	agentgrpc "github.com/DataDog/datadog-agent/pkg/util/grpc"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+func init() {
+	AgentCmd.AddCommand(remoteConfigCmd)
+}
+
+var remoteConfigCmd = &cobra.Command{
```

Comment:
If this is not intended to be publicly documented etc. yet, we should consider `Hidden: true` here.

Reply:
Good idea, I've just added `Hidden: true` 👍